site2/docs/functions-runtime.md

```diff
@@ -183,3 +249,68 @@ for the `runtimeCustomerClassName` property. This implementation takes the follo
   }
 }
 ```
+
+## Other configuration considerations
+
+
+### In a cluster with geo-replication
+
+If you are running multiple clusters tied together with geo-replication, it is important to use a different function namespace for each cluster! Otherwise, the functions
```

Comment:
```suggestion
If you are running multiple clusters tied together with geo-replication, it is important to use a different function namespace for each cluster. Otherwise, the function
```

IBM Writing Style Guide: 
Do not punctuate sentences with exclamation points because their tone can be interpreted negatively,
for example, as aggressive, condescending, or overly informal.
Convey urgency or emphasis with the appropriate words, not with exclamation points. To call
attention to important hints, tips, guidance, restrictions, or advice that might be overlooked, consider
using a note that has a meaningful label.